src/DevChatter.DevStreams.Infra.Dapper/Crud/Crud.tt

```diff
@@ -0,0 +1,216 @@
+﻿<#@ assembly name="System.Core" #>
```

Comment:
So I hacked this in 10 min but basically this allows you define a model of the data types in T4. From this we generate the Data classes as well as the CRUD ops.

Reply:
I haven't coded C# for years so I might have missed things.